paymentsheet/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -87,7 +88,9 @@ internal abstract class BaseAddPaymentMethodFragment(
             setupRecyclerView(viewBinding, paymentMethods)
         }
 
-        replacePaymentMethodFragment(paymentMethods[0])
+        if (paymentMethods.isNotEmpty()) {
```

Comment:
Should we return an error early here? If paymentMethods is empty there's nothing to do, right?

Reply:
The return, onFatal is called before BaseAddPaymentMethodFragment really can do anything. It might be better to pass the StripeIntent to the BaseAddPaymentMethodFragment rather than using the BaseSheetViewModel, but I think that is a bigger design that could use more thinking through.